suitcase/mongo_normalized/__init__.py

```diff
@@ -90,6 +92,27 @@ def __call__(self, name, doc):
         sanitized_doc = event_model.sanitize_doc(doc)
         return super().__call__(name, sanitized_doc)
 
+    def update(self, name, doc):
+        if name == 'start':
+            schema_validators[DocumentNames.start].validate(doc)
+            current_col = self._run_start_collection
+            revisions_col = self._run_start_collection_revisions
+            old = current_col.find_one({'uid': doc['uid']})
+            old.pop('_id')
+            target_uid_docs = revisions_col.find({'uid': doc['uid']})
+            cur = target_uid_docs.sort([('revision', pymongo.DESCENDING)]).limit(1)
+            wrapped = dict()
+            try:
+                wrapped['revision'] = next(cur)['revision'] + 1
+            except StopIteration:
+                wrapped['revision'] = 0
+            wrapped['document'] = old
+            wrapped['uid'] = wrapped['document']['uid']
```

Comment:
Per my comment above, I think we can delete this line. (I know I suggested in the first place. I was momentarily forgetting that Mongo can query on nested keys.)